docs/DeveloperGuide.md

```diff
@@ -2,37 +2,254 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
-
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+[comment]: <> (//@@author Jeraldchen)
+
+### User Interface
+![UIClass.png](diagrams%2FUIClass.png)
+
+This feature shows how the user interface is being implemented via three main classes which are Menu, Information and Parser.
+
+### Menu
+The menu class is responsible for the user interface of the application. It is responsible for displaying the menu and
+taking in user input. The menu class is also responsible for the main loop of the application, which is the loop that
+continues to run until the user exits the application. The menu class allows patients to log in, register and exit the
+program. 
+
+`showWelcomeMenu()` is used to display the welcome menu to the user.
+`register()` is used to register a new patient into the system.
+`login()` is used to log a patient into the system.
+`exit()` is used to exit the program.
+`showAccountMenu()` is used to display the list of choices to the user.
+`getUserSymptoms` is to ask input form user and store it in an arraylist.
```

Comment:
Minor issue here, getUserSymptoms should have brackets after it as seen in the user interface diagram. Overall, great job with the formatting of the DG, which looks very neat